src/patternfly/demos/Card/examples/CardView.md

```diff
@@ -284,3 +286,1028 @@ wrapperTag: div
+      {{#> card}}
+        {{#> card-title card-title--modifier="pf-u-text-align-center"}}
+            50 Hosts
+        {{/card-title}}
+        {{#> card-body card-body--modifier="pf-u-text-align-center"}}
+          {{#> l-flex l-flex--modifier="pf-m-inline-flex"}}
+            {{#> l-flex l-flex--modifier="pf-m-space-items-sm"}}
+              {{#> l-flex-item}}
+                <i class="fas fa-check-circle pf-u-success-color-100" aria-hidden="true"></i>
+              {{/l-flex-item}}
+              {{#> l-flex-item}}
+                <a href="#">5</a>
+              {{/l-flex-item}}
+            {{/l-flex}}
+            {{> divider divider--modifier="pf-m-vertical"}}
+            {{#> l-flex l-flex--modifier="pf-m-space-items-sm"}}
+              {{#> l-flex-item}}
+                <i class="fas fa-times-circle pf-u-danger-color-100" aria-hidden="true"></i>
+              {{/l-flex-item}}
+              {{#> l-flex-item}}
+                <a href="#">12</a>
+              {{/l-flex-item}}
+            {{/l-flex}}
+          {{/l-flex}}
+        {{/card-body}}
+      {{/card}}
+      {{#> card}}
+        {{#> card-title card-title--modifier="pf-u-text-align-center"}}
+            12 Hosts
+        {{/card-title}}
+        {{#> card-body card-body--modifier="pf-u-text-align-center"}}
+          {{#> l-flex l-flex--modifier="pf-m-inline-flex"}}
+            {{#> l-flex l-flex--modifier="pf-m-space-items-sm"}}
+              {{#> l-flex-item}}
+                <i class="fas fa-exclamation-triangle pf-u-warning-color-100" aria-hidden="true"></i>
+              {{/l-flex-item}}
+              {{#> l-flex-item}}
+                <a href="#">2</a>
+              {{/l-flex-item}}
+            {{/l-flex}}
+            {{> divider divider--modifier="pf-m-vertical"}}
+            {{#> l-flex l-flex--modifier="pf-m-space-items-sm"}}
+              {{#> l-flex-item}}
+                <i class="fas fa-times-circle pf-u-danger-color-100" aria-hidden="true"></i>
+              {{/l-flex-item}}
+              {{#> l-flex-item}}
+                <a href="#">7</a>
+              {{/l-flex-item}}
+            {{/l-flex}}
+          {{/l-flex}}
+        {{/card-body}}
+      {{/card}}
+    {{/gallery}}
+  {{/grid-item}}
+  {{#> grid-item}}
+    {{#> gallery gallery--modifier="pf-m-gutter" gallery--attribute='style="--pf-l-gallery--GridTemplateColumns--min: 260px;"'}}
+      {{#> card}}
+        {{#> card-title card-title--modifier="pf-u-text-align-center"}}
+            13 Hosts
+        {{/card-title}}
+        {{#> card-body card-body--modifier="pf-u-text-align-center"}}
+          {{#> l-flex l-flex--modifier="pf-m-justify-content-space-around"}}
+            {{#> l-flex newcontext}}
+              {{#> l-flex-item}}
+                <i class="fas fa-times-circle pf-u-danger-color-100" aria-hidden="true"></i>
+              {{/l-flex-item}}
+              {{#> l-flex l-flex--modifier="pf-m-column pf-m-align-items-flex-start pf-m-space-items-none"}}
```

Comment:
if you remove text-align-center from this parent, you can remove pf-m-align-items-flex-start, too